lib/gcloud/resource_manager/project.rb

```diff
@@ -359,6 +359,153 @@ def undelete
       end
 
       ##
+      # Gets the access control policy. Read
```

Comment:
The term 'access control policy' is great, and I like the link to the managing guide, but in order to make things instantly clear to a user in a hurry, it would be good to sprinkle 'IAM' and possibly 'Cloud IAM' over these docs. Possibly with a link to [https://cloud.google.com/iam/](https://cloud.google.com/iam/).